test/org/sagebionetworks/bridge/dynamodb/DynamoCriteriaDaoTest.java

```diff
@@ -0,0 +1,79 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import javax.annotation.Resource;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import org.sagebionetworks.bridge.models.Criteria;
+
+import com.google.common.collect.Sets;
+
+@ContextConfiguration("classpath:test-context.xml")
+@RunWith(SpringJUnit4ClassRunner.class)
+public class DynamoCriteriaDaoTest {
```

Comment:
Is it worth adding a test to verify that we can create/update from a ScheduleCriteria or a Subpopulation?